accounts/models.py

```diff
@@ -1,3 +1,38 @@
 from django.db import models
 
-# Create your models here.
+from common.models import TimeStampedModel
+from users.models import User
+
+
+class Account(TimeStampedModel):
+    number = models.CharField(max_length=16, unique=True)
+    is_debit = models.BooleanField(default=True)
+    CVV = models.IntegerField()
+    pincode = models.IntegerField()
+    expiration_date = models.DateField()
+    amount = models.FloatField(null=True)
+    credit_limit = models.FloatField(null=True)
+    deposit_amount = models.FloatField(null=True)
+    credit_percentage = models.FloatField(null=True)
+    deposit_percentage = models.FloatField(null=True)
+    credit_start_date = models.DateTimeField(null=True)
+    credit_end_date = models.DateTimeField(null=True)
+    deposit_start_date = models.DateTimeField(null=True)
+    deposit_end_date = models.DateTimeField(null=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+
+
+class Transaction(TimeStampedModel):
+    account = models.ForeignKey(Account, on_delete=models.CASCADE)
+    type_of_expence = models.CharField(max_length=50)
```

Comment:
maybe we should add choices for this field

Reply:
> maybe we should add choices for this field

what do you mean, could you please explain more?

Reply:
@olyapetryshyn, can you provide examples?

Reply:
https://docs.djangoproject.com/en/3.0/ref/models/fields/

Reply:
read about enumeration types

Reply:
> read about enumeration types

I got it, but which choices you want to see here. We didn't speak about that.